templates/index.html

```diff
@@ -10,9 +10,53 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+      <form action="/update" method="POST">
```

Comment:
If you moved this form inside the for loop you would be able to update the status with a click of a submit button.

This would also make the logic in app.py a little cleaner and make adding additional statuses possibl.

Reply:
Not sure I follow. 

The statuses **are** updated by the click of a submit button. By having the `form` outside the loop it has access to all elements which means I can update multiple fields at once, hence the `for` loop in `app.py`s `update()` method.

Additional statuses could be added to my solution with another column of checkboxes though if we're adding many types of statuses then a drop down would probably be the best approach.